geomesa-z3/src/main/scala/org/locationtech/geomesa/curve/Z3SFC.scala

```diff
@@ -23,9 +23,9 @@ class Z3SFC(period: TimePeriod, precision: Int = 21) extends SpaceTimeFillingCur
 
   require(precision > 0 && precision < 22, "Precision (bits) per dimension must be in [1,21]")
 
-  override val lon: NormalizedDimension  = NormalizedLon(precision)
-  override val lat: NormalizedDimension  = NormalizedLat(precision)
-  override val time: NormalizedDimension = NormalizedTime(precision, BinnedTime.maxOffset(period).toDouble)
+  val lon: NormalizedDimension  = NormalizedLon(precision)
+  val lat: NormalizedDimension  = NormalizedLat(precision)
+  val time: NormalizedDimension = NormalizedTime(precision, BinnedTime.maxOffset(period).toDouble)
```

Comment:
As a sanity check, we are tossing lon/lat/time out of the common interface, but Z2/Z3 still have them in use?

Reply:
right. does that seem reasonable? we do use them directly in the Z2/Z3 feature index code. but the s2 index doesn't have them so it seemed more an implementation detail.

Reply:
Works for me.